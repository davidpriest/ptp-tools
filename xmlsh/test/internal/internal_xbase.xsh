# test of xbase
xbase < http://test.xmlsh.org/data/books.xml
X=$<(xaddbase < http://test.xmlsh.org/data/books.xml)
X2=<[ $X/BOOKLIST ]>
xbase <{X2}