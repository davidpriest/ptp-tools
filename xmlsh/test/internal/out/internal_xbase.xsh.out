http://test.xmlsh.org/data/books.xml
http://test.xmlsh.org/data/books.xml
